docs/build.adoc

```diff
@@ -363,9 +363,12 @@ ____
 We use this environment in the CI https://github.com/KhronosGroupActions/android-ndk-build[Android NDK Repository]
 ____
 
-It is highly recommended to install https://d.android.com/studio[Android Studio] to build, run and trace the sample project.
-Android Studio uses the following plugins/tools to build samples:
+=== Android Studio (Recommended)
+
+It is highly recommended to install https://d.android.com/studio[Android Studio] to build, run and trace the sample project. The samples require Android Studio Ladybug.
```

Comment:
Do we want to lock it down to Ladybug only?  Might be better to simply revert that back to highly recommended to install...  Also, the samples, don't strictly require Android Studio.  I can build / install / run them with just the tools; actually, that's what CI is doing come to think of it.

Reply:
Yes, LadyBug is currently the only supported Android Studio Version, all others are EOL. Building from CLI still works fine, but most people (see issues) use Android Studio anyway.

Reply:
fair enough, just saying "required" is a strong word ;-)

Reply:
Totally agree, will adjust wording there to not confuse people :)

Reply:
Changed wording a bit to make it clear that Ladybug is a requirement only when building directly through Android Studio. Hope that's better :)

Reply:
Perfect thanks